src/puppet-web/event.ts

```diff
@@ -244,6 +244,10 @@ async function onServerLogin(this: PuppetWeb, data, attempt = 0): Promise<void>
   // if `login` event fired before this.bridge inited, we delay the event for 1 second.
   if (!this.bridge) {
     log.verbose('PuppetWebEvent', 'onServerLogin() fired before bridge inited. delay for 1 second.')
+    if (!arguments) {
+      log.warn('PuppetWebEvent', 'onServerLogin() arguments is null')
+      throw new Error('onServerLogin() arguments is null')
+    }
     setTimeout(() => {
       onServerLogin.apply(this, arguments)
```

Comment:
`onServerLogin.apply(this, args)`

We can not use `arguments` directly because it's not a **real** array.

See: https://stackoverflow.com/a/676728/1123955